components/ImageCropper/src/Dependencies.props

```
@@ -11,7 +11,7 @@
 <Project>
     <!-- WinUI 2 / UWP -->
     <ItemGroup Condition="'$(IsUwp)' == 'true'">
-        <PackageReference Include="Win2D.uwp" Version="1.25.0"/>
+        <PackageReference Include="Win2D.uwp" Version="1.26.0"/>
```

Why 1.26? The latest for UWP is 1.28.0.

---

Looks like it's been updated since this PR. Let's bump it again.  
```suggestion
        <PackageReference Include="Win2D.uwp" Version="1.28.0"/>
```